src/Components/Web.JS/src/Services/NavigationManager.ts

```diff
@@ -152,6 +180,13 @@ async function performInternalNavigation(absoluteInternalHref: string, intercept
   // we render the new page. As a best approximation, wait until the next batch.
   resetScrollAfterNextBatch();
 
+  const hashIndex = absoluteInternalHref.indexOf('#');
+  // Excluding cases when hash="#"
+  if ( hashIndex > -1 && absoluteInternalHref.length > hashIndex + 1) {
+    var identifier = absoluteInternalHref.substring(hashIndex+1);
+    setTimeoutToScrollToElement(identifier);
+  }
```

Comment:
Minor code style issues: leading space after `(` on L185, use `const` instead of `var`

Reply:
Also inconsistent spacing around `+`